plugins/cloud-aws/rest-api-spec/test/cloud_aws/20_repository.yaml

```diff
@@ -0,0 +1,23 @@
+# Integration tests for Cloud AWS components
+#
+"S3 repository can be registereed":
+    - do:
+        snapshot.create_repository:
+          repository: test_repo_s3_1
+          verify: false
+          body:
+            type: s3
+            settings:
+              bucket: "my_bucket_name"
+              access_key: "AKVAIQBF2RECL7FJWGJQ"
+              secret_key: "vExyMThREXeRMm/b/LRzEB8jWwvzQeXgjqMX+6br"
```

Comment:
These keys were copied from the [aws plugin documentation](https://github.com/elastic/elasticsearch/tree/master/plugins/cloud-aws#generic-configuration). If they were real, we would be probably filing for bankruptcy by now.